### DISCUSSION TITLE
7月14日铁道兑换测试

### DISCUSSION THREAD

**Discussion by nyano** [Ljzd-PRO/Mys_Goods_Tool#142] [Q&A]
1.自己改了V1版本兑换，日志如下：
[mys_goods_tool.log](https://github.com/Ljzd-PRO/Mys_Goods_Tool/files/12048621/mys_goods_tool.log)
日志是我自己复制进去的，程序没写进去
总结：1028
我的V1日志没写到子目录下的log目录，而是写到上级目录v2的log里面去了么？

2.同时开v2版本
[mys_goods_tool.log](https://github.com/Ljzd-PRO/Mys_Goods_Tool/files/12048656/mys_goods_tool.log)

总结：2106

3.同时，手动米游社兑换
兑换成功
![c7bb36ba9f24c743b8e3316a0bda777](https://github.com/Ljzd-PRO/Mys_Goods_Tool/assets/5079651/73888fb4-145b-4a27-8a9f-ba9e1ba5b272)

未触发验证码。所以2106和1028可能不是验证码造成的问题

这个GUI能不能换个库。。。

到底哪里看v2版本的版本号？日志里写的是1.4.4？

**Comment by ayakasuki** [comment]
能不能进入兑换模式后 意外关闭GUI，点进去不用自己再次确定兑换模式呢。。 我一不小心没留意 只看到下面地址什么选上以为是OK了。。。 04分才看到 半个月抢立牌这次无了QAQ

**Comment by nyano** [comment]
> 能不能进入兑换模式后 意外关闭GUI，点进去不用自己再次确定兑换模式呢。。 我一不小心没留意 只看到下面地址什么选上以为是OK了。。。 04分才看到 半个月抢立牌这次无了QAQ

主要还是GUI太难用了。鼠标位置和实际位置不符 ，错屏，然后还重复显示等等
同楼上，v1版本进入main.py后无需其他按键，V2版本跟不上节奏，就回首先要想办法拉到最下面，然后还得鼠标能点的上进入模式，就很慢了。
测试还要去log里面看结果等等。
目前来说，V2相比V1唯一的好处是不需要手动改config了，但是目的应该是尽可能简便的兑换才对

**Comment by Ljzd-PRO** [comment]
主要是终端图形界面增加了鼠标支持，这不太稳定。如果是靠键盘上下左右回车和Tab交互的，是不会有问题的。
如果兑换的问题能解决，之后会加上这种模式
